static/src/stylesheets/module/content/_live-blog.scss

```diff
@@ -271,6 +271,27 @@ $block-padding-right: $gs-gutter;
 
     .block-elements {
 
+        /* Bylines
+           ========================================================= */
+        .liveblog-block-byline {
+            border-bottom: 1px dotted #dcdcdc;
+            padding-bottom: 8px;
```

Comment:
Should generally do vertical margins as calculations on $gs-baseline and horizontal margins as calculations on $gs-gutter

See https://github.com/guardian/guss-grid-system